jax/experimental/jax2tf/jax2tf.py

```diff
@@ -1287,9 +1287,10 @@ def _conv_general_dilated(lhs, rhs, window_strides, padding, lhs_dilation,
 tf_impl_with_avals[lax.conv_general_dilated_p] = _conv_general_dilated
 
 
-def _dot_general(lhs, rhs, dimension_numbers, precision):
+def _dot_general(lhs, rhs, dimension_numbers, precision, preferred_element_type):
   """Implementation of lax.dot_general_p in terms of tf.linalg.einsum."""
   del precision
+  del preferred_element_type
```

Comment:
I assume that `tf.linalg.einsum`, which we use to convert the `lax.dot_general_p`, does not support this capability?

Reply:
It unfortunately doesn't at the moment. We're starting to look into it what it would take to add this on the Tensorflow side.